pkg/server/debug.go

```diff
@@ -41,11 +40,22 @@ const debugEndpoint = "/debug/"
 // register to that via import, and go-metrics registers to that via exp.Exp())
 var debugServeMux = http.DefaultServeMux
 
-// TODO(arjun): use an Enum setting here.
-var debugRemote = settings.RegisterStringSetting(
```

Comment:
Going forward, in-place type changes are usually a Bad Idea as they'll make deserialization angry, but probably fine pre-1.0.